Dockerfile.prod

```diff
@@ -0,0 +1,19 @@
+# Stage 1: Typescript build app
+FROM node:16-alpine AS ts-build
+WORKDIR /usr/app
+COPY package.json .
+RUN npm install
+COPY . .
+RUN npm run ts:build
```

Comment:
I dont entirely understand whats going on , but above u have `ts-build` and here u have `ts:build`
Other than that, LGTM

Reply:
`ts:build` is an npm/yarn script we have in `package.json`, and `ts-build` above is just a label for this stage in the Dockerfile. We can pair and I can walk you through this if you want.